devops/azure-pipelines.yml

```diff
@@ -79,3 +79,28 @@ stages:
 
   displayName: Dockerize --> Stage
 
+- stage: deploy
+  dependsOn:
+  - analyze
+  - dockerize
+  jobs:
+
+  - job: deploy
+
+    steps:
+    - checkout: templates
+
+    - template: ./templates/deploy.yml@templates
+      parameters:
+        containerRegistry: 'acr-team1'
+        devCluster: 'k8s-team1-config-dev'
+        stagingCluster: 'k8s-team1-config-staging'
+        devBranch: 'branch-dev'
+        stagingBranch: 'main'
+        namespace: 'team1'
+        manifest: 'k8s/microservices/gateway.yml'
```

Comment:
https://github.com/1053-August-Duet-Project-Registry/project-registry-devops/pull/11#discussion_r677125419
but if there's not enough time to change it before showcase, then... do what you think is best as a team

Reply:
I THINK we got this covered, since each microservice is updating it's own image, and this is just because the k8s manifest moved.